source/contributor/faqs.rst

```diff
@@ -169,3 +169,67 @@ which elements and values can appear in ``meta.yaml``.
 
 Conda has this information available `here <https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html>`_.
 Please check that you are looking at the correct version of the documentation for the current conda version used by bioconda. 
+
+
+.. _platform-nomenclature-faq:
+
+Understanding platform nomenclature
+-----------------------------------
+
+Different CPU chips use different architecture, so programs are written
+fundamentally differently for them. Why do we care about this for conda
+packages? Because a package with compiled dependencies must have
+platform-specific dependencies.
+
+New Apple Silicon 
+
+There is a lot of confusing nomenclature surrounding them. Here is an attempt
+at clearing them up, or at least providing enough context that you can look up
+more details on your own:
+
+**instruction set, CISC, RISC, RISC-V**: The *instruction set* is the assembly
+code commands that are possible for the chip. *CISC* is "complex instruction set
+computer", prioritizing flexibility; *RISC* is "reduced instruction set
+computer", prioritizing power consumption (oversimplification, but that's the
+general idea). Instruction sets can be proprietary. ARM is a company that
+licenses a widely-used proprietary reduced instruction set. RISC-V is an open
+(non-proprietary) reduced instruction set.
+
+**ARM vs ARM RISC:** ARM is a company. They make chips (for example, the ones
```

Comment:
I think ARM designs, not necessarily makes the chips

Reply:
Thanks, yep this has been addressed (https://github.com/bioconda/bioconda-docs/pull/16/files#diff-5a476a141b5805f24f43cab6c3548f1dd1319013abd9035ef91062a6fcd9a35bR543-R558)